test/shared/src/main/scala/zio/test/package.scala

```diff
@@ -20,7 +20,7 @@ import scala.deprecated
 
 import zio.duration.Duration
 import zio.stream.{ ZSink, ZStream }
-import zio.test.environment.{ TestClock, TestConsole, TestRandom, TestSystem }
+import zio.test.environment.{ testEnvironmentManaged, TestClock, TestConsole, TestEnvironment, TestRandom, TestSystem }
```

Comment:
Maybe wildcard import at this point?

Reply:
I am not a big fan of wildcard. This sounds like something to add to scalafix, if it is doable. I typically put wildcard when it passes the line limit. Scalafmt can also just put one import per line.

Reply:
Sounds good.